larpandoracontent/LArMonitoring/TwoViewTransverseTracksValidationTool.cc

```diff
@@ -0,0 +1,164 @@
+/**
+ *  @file   larpandoracontent/LArMonitoring/TwoViewTransverseTracksValidationTool.cc
+ *
+ *  @brief  Implementation of the two view transverse tracks validation tool.
+ *
+ *  $Log: $
+ */
+#include "Helpers/MCParticleHelper.h"
+
+#include "larpandoracontent/LArMonitoring/TwoViewTransverseTracksValidationTool.h"
+
+#include <limits>
+
+using namespace pandora;
+
+namespace lar_content
+{
+
+TwoViewTransverseTracksValidationTool::TwoViewTransverseTracksValidationTool() :
+    m_treeName("mytree"), 
+    m_outputFileName("output.root")
+{
+}
+
+TwoViewTransverseTracksValidationTool::~TwoViewTransverseTracksValidationTool()
+{
+    PANDORA_MONITORING_API(SaveTree(this->GetPandora(), m_treeName.c_str(), m_outputFileName.c_str(), "UPDATE"));
```

Comment:
Do we want tree writing to be optional here?

Reply:
The validation tool, as it is, is supposed to make a TTree by design.  As we look to move the base functionality out (such as moving the TTree data registration to the SDK, for example) then we may want to revisit this question.

Reply:
(This'll go in PandoraMonitoring, typically released alongside the SDK as a single Pandora external product, with two libraries.)